main.py

```
@@ -82,25 +82,29 @@ async def status():
 
 
 @app.post("/query", summary="Start a Research Task")
-async def query(query: str = Form(...), file: UploadFile = File(None)):
+async def query(query: str = Form(...), files: list[UploadFile] = File(None), session_id: str = Form(...)):
```

Ruff (0.11.9)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)